chia/wallet/vault/vault_drivers.py

```diff
@@ -45,3 +46,8 @@ def get_vault_proof(merkle_tree: MerkleTree, puzzlehash: bytes32) -> Program:
     proof = merkle_tree.generate_proof(puzzlehash)
     vault_proof: Program = Program.to((proof[0], proof[1][0]))
     return vault_proof
+
+
+# SECP SIGNATURE
+def construct_secp_message(delegated_puzzlehash: bytes32, coin_id: bytes32) -> bytes:
+    return delegated_puzzlehash + coin_id + DEFAULT_CONSTANTS.GENESIS_CHALLENGE
```

Comment:
You shouldn't be using this default here.  At the very least make it a default kwarg, but I don't recommend that either.

Reply:
moved it out to the test